src/JestProcessManagement/JestProcess.ts

```diff
@@ -68,9 +68,12 @@ export class JestProcess {
     this.stopResolveCallback = null
     let exited = false
 
+    const reporterPath = `${process.cwd()}/src/JestProcessManagement/vscode-jest-reporter.ts`
```

Comment:
this didn't seem to work:`process.cwd()` returns "/"

Reply:
Do you have any idea how to reference extension directory in vscode? This works in my case probably because of running in development mode.

Reply:
[extensionPath](https://code.visualstudio.com/api/references/vscode-api#Extension) seems promising...?

Reply:
Thanks, I've just used `extensionName` instead of `extensionId` which obviously doesn't work as expected.